idea/src/org/jetbrains/kotlin/idea/versions/KotlinRuntimeLibraryUtil.java

```diff
@@ -63,12 +66,36 @@ private KotlinRuntimeLibraryUtil() {}
 
     @NotNull
     public static Collection<VirtualFile> getLibraryRootsWithAbiIncompatibleKotlinClasses(@NotNull Project project) {
-        return getLibraryRootsWithAbiIncompatibleVersion(project, KotlinAbiVersionIndex.INSTANCE$);
+        return getLibraryRootsWithAbiIncompatibleVersion(project, KotlinAbiVersionIndex.INSTANCE$,
+                            new Predicate<Module>() {
+                             @Override
+                             public boolean apply(@Nullable Module module) {
+                                 return module != null && ProjectStructureUtil.isJavaKotlinModule(module);
+                             }
+                            },
+                            new Predicate<Integer>() {
+                             @Override
+                             public boolean apply(Integer abiVersion) {
+                                 return !AbiVersionUtil.isAbiVersionCompatible(abiVersion);
+                             }
+                            });
```

Comment:
Just add a newline between `(` and `project` and press Ctrl+Shift+L / Cmd+Shift+L